seedfarmer/__main__.py

```diff
@@ -57,6 +57,12 @@ def version() -> None:
     help="The AWS region to use for boto3.Sessions",
     required=False,
 )
+@click.option(
+    "--qualifier",
+    default=None,
+    help="A qualifier to append to toolchain role",
```

Comment:
Are you appending qualifier only to toolchain? Not to the relevant deployment roles? If to both, the description should be generic

Reply:
it is being appended to toolchain role...that text is off